Misc/NEWS.d/next/Build/2024-08-02-12-59-15.gh-issue-118943.vZQtET.rst

```diff
@@ -0,0 +1,2 @@
+The ``regen-jit`` make target is no longer part of ``regen-all`` to avoid various
+race conditions when running ``regen-all``.
```

Comment:
I'd rephrase this, since we still regenerate the JIT, just once instead of twice. Something like this:

```suggestion
Fix an issue where the experimental JIT could be built several times by the ``make regen-all`` target, leading to possible race conditions on heavily parallelized builds.
```